game/opengl_text.cc

```diff
@@ -246,22 +171,25 @@ void SvgTxtTheme::draw(Window& window, std::string _text) {
 
 void SvgTxtTheme::draw(Window& window, std::vector<TZoomText>& _text, bool lyrics) {
 	std::string tmp;
-	for (auto& zt: _text) { tmp += zt.string; }
+
+	for (auto& zt: _text)
+		tmp += zt.string;
+
 	if (m_opengl_text.size() != _text.size() || m_cache_text != tmp) {
 		m_cache_text = tmp;
 		m_opengl_text.clear();
+		auto renderer = TextRenderer();
 		for (auto& zt: _text) {
-			m_text.text = zt.string;
-			auto openGlPtr = std::unique_ptr<OpenGLText>(std::make_unique<OpenGLText>(m_text, m_factor));
+			auto openGlPtr = std::make_unique<OpenGLText>(renderer.render(zt.string, m_textstyle, m_factor));
```

Comment:
we're making a lot of unique smart pointers here. Shouldn't we make the instance first, fill it and push it back to the vector and then clearing it and repeating the process of filling/inserting in the vector/clear?

Reply:
Why? I didn't get the point ... sorry